### PR TITLE
Accessibility popup close button

### PR DESCRIPTION
**Pull request by Falke-Design**
- Change the color of the close button for a better contrast #7539 
- Add `role=button` and the `aria-label='Close popup'` to the `<a>` element #7399
- Add `aria-hidden` to the `times` / `x` character to prevent the reading of screen readers #7472

Close Button old:
grafik
New:
grafik

Fixes: #7539, #7399, #7472

Related PR: #7079
@Malvoz fyi

### CODE REVIEW

**Review by Malvoz — APPROVED** [review]
Nice, LGTM!

File: dist/leaflet.css [hunk 1]
```diff
@@ -523,13 +523,13 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	width: 18px;
 	height: 14px;
 	font: 16px/14px Tahoma, Verdana, sans-serif;
-	color: #c3c3c3;
+	color: #757575;
 	text-decoration: none;
 	font-weight: bold;
```
Comment (mourner) [inline dist/leaflet.css:528]:
```suggestion
```

**Review by mourner — CHANGES_REQUESTED** [review]
This looks great! I find the close button too jarring when simpling making the color darker, but not if we remove the bold font:

image

**Review by mourner — APPROVED** [review]

**Comment by ahukkanen** [comment]
The close button is still not accessible for other languages than English because the label cannot be translated.

**Comment by Malvoz** [comment]
@ahukkanen https://github.com/Leaflet/Leaflet/issues/3210#issuecomment-634066856:

> Regarding translations: [internationalization isn't built into leaflet core](https://github.com/Leaflet/Leaflet/pull/1232) but it is available as a plugin - https://github.com/yohanboniface/Leaflet.i18n

I suppose you could open a separate issue, or ask if the maintainers could reconsider https://github.com/Leaflet/Leaflet/pull/1232, but that's off topic from this PR?

**Comment by ahukkanen** [comment]
@Malvoz Thanks for the context!

After reading that i18n discussion, I'm not too certain I could convince the maintainers otherwise than what was discussed there.

At https://github.com/Leaflet/Leaflet/pull/7079#issuecomment-634080439 I tried to suggest to make the hard coded texts configuration options instead of hard coded. This way it would be possible to change programmatically without the need of creating a separate i18n layer in the core.

**Pull request merged**